src/main/java/aisu/Aisu.java

```diff
@@ -0,0 +1,93 @@
+package aisu;
+
+import aisu.command.Command;
+import aisu.exception.AisuException;
+import aisu.parser.Parser;
+import aisu.storage.Storage;
+import aisu.task.Task;
+import aisu.tasklist.TaskList;
+import aisu.ui.Ui;
+
+import java.util.List;
+
+
+/**
+ * A tasklist chatbot, named Aisu.
+ *
+ * @author Liaw Xin Yan
+ */
+public class Aisu {
+    private final Storage storage;
+    private TaskList tasklist;
+    private final Ui ui;
```

Comment:
Ordering of fields can be standardised, e.g. private final fields before private fields as such:
```suggestion
    private final Storage storage;
    private final Ui ui;
    private TaskList tasklist;
```